spec/services/reinstate_declined_offer_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.describe ReinstateDeclinedOffer, with_audited: true do
+  describe '#save!' do
+    it 'updates the application course choice status back to "offer made" and resets the DBD days back to 10' do
+      Timecop.freeze do
+        course_choice = create(:application_choice, status: :declined)
+        zendesk_ticket = 'www.becomingateacher.zendesk.com/agent/tickets/example'
+
+        ReinstateDeclinedOffer.new(course_choice: course_choice, zendesk_ticket: zendesk_ticket).save!
+
+        expect(course_choice).to have_attributes({
+          status: 'offer',
+          declined_at: nil,
+          declined_by_default: false,
+          decline_by_default_at: Time.zone.now + 10.days,
+        })
```

Comment:
It would be nice not to have to repeat this out here. Could this spec take a choice in the "offer" state, give it to the `DeclineOffer` service, then give it to the new service? Then compare the original with the reverted?